source/vocab/agents.ttl

```diff
@@ -277,6 +277,8 @@
     owl:equivalentClass sdo:Library .
 
 :bibliography a owl:ObjectProperty ;
+    :category :compositional ;
```

Comment:
This is not logically compositional. We should not "hack" the vocab for interface needs like this (it will just shadow the lack of linkable bibliographies).